### PR TITLE
Fix/load bot max conns per host

### PR DESCRIPTION
**Pull request by munna0908**
# Description

This PR adds a new flag to configure `MaxConnsPerHost` in loadbot. 
As part of this PR, I have also created a PR on our vendor repo go-web3, here's is the [link](https://github.com/umbracle/go-web3/pull/125). 
Docs [PR](https://github.com/0xPolygon/polygon-sdk-docs/pull/40)

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have tested this code
- [x] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

**Review by lazartravica — APPROVED** [review]

**Review by zivkovicmilos — APPROVED** [review]
Looks great 💯 

@munna0908 
Please add the relevant docs changes as well 👍

File: command/loadbot/loadbot_command.go [hunk 1]
```diff
@@ -96,14 +96,16 @@ func (l *LoadbotCommand) Run(args []string) int {
 	var valueRaw string
 	var count uint64
 	var jsonrpc string
-
+	var maxConns int
+	defaultMaxConns := int(2 * tps)
```
Comment (Kourin1996) [inline command/loadbot/loadbot_command.go]: this will be zero because `tps` is not set yet

**Review by Kourin1996 — CHANGES_REQUESTED** [review]

**Review by brkomir — APPROVED** [review]
Apart from comments left by other reviewers, nothing to add!

File: command/loadbot/loadbot_command.go (hunk 1 above)
Comment (munna0908) [reply command/loadbot/loadbot_command.go]: My Bad, Fixed it commit [link](https://github.com/0xPolygon/polygon-sdk/pull/295/commits/22bf32d3e2e575e6582e0d2f2325e4fbfca478d3)

Comment (Kourin1996) [reply command/loadbot/loadbot_command.go]: @munna0908 Hi, please note the value will be set by `flags.Parse(args)`.

Comment (munna0908) [reply command/loadbot/loadbot_command.go]: Fixed it commit-[link](https://github.com/0xPolygon/polygon-sdk/pull/295/commits/5511eda215fb0f798230c10ad91357a698ae37d7)

**Review by Kourin1996 — APPROVED** [review]
LGTM 👌

**Pull request merged**